lib/ImageHelper.php

```diff
@@ -367,7 +367,10 @@ private static function analyze_url( $url ) {
 			}
 		} else {
 			if ( !$result['absolute'] ) {
-				$tmp = site_url().$tmp;
+				$pos = strpos(content_url(), CONTENT_DIR);
```

Comment:
Not in this case, CONTENT_DIR is a constant defined by the bedrock code which I didn't realize isn't core wordpress. I've changed it now.